drivers/reset/reset_rpi_pico.c

```diff
@@ -1,122 +1,35 @@
 /*
- * Copyright (c) 2022 Andrei-Edward Popa
+ * Copyright (c) 2025 Dmitrii Sharshakov <d3dx12.xx@gmail.com>
```

Comment:
Please do not erase the names of past author.

Reply:
It's a full rewrite, so I thought the author of the original is not in charge of the current version. I do not object to keeping their copyright as well, if that is still applicable

Reply:
There are even functions with no diff at all, so I think that's enough to keep the original author's name.

Reply:
Yea, agreed